shifthelper/checks/__init__.py

```diff
@@ -115,15 +115,27 @@ def __init__(self,
     def run(self):
         while not self.stop_event.is_set():
             if len(self.queue) > 0:
```

Comment:
The check is for the calling. We call if any message is in the queue and the messenger sends a notification for each message.